app/views/layouts/cms/_resource.html.erb

```diff
@@ -15,10 +15,12 @@
     <% unless get_resource_links.empty? %>
       <ul class="list--links">
         <% get_resource_links.each do |link| %>
-          <li class="list__li">
-            <span><%= link[:text] %></span>
-            <%= link_to link[:button], link[:url], title: link[:text], class: "list__a #{link[:classes]}", target: '_blank' %>
-          </li>
+          <% unless link[:url].nil? || link[:url].blank? %>
```

Comment:
`.blank?` already accounts for nil values as well, so the first bit can be omitted